src/interfaces/token.spec.ts

```diff
@@ -766,16 +767,16 @@ describe("TokenInterface", () => {
     });
   });
 
-  describe("approve", () => {
-    beforeEach(() => {
-      sendTransactionMock.mockResolvedValue(true);
-    });
-
-    it("should return a transaction response when approving non native token", async () => {
-      approveMock.mockReturnValue("approved");
-      const approveResult = await tokenInterface.approve(ownerAddress, tokenAddress, spenderAddress, amount);
+  describe("populateApprove", () => {
+    it("should return a populated transaction when approving non native token", async () => {
+      const populateApproveResult = await tokenInterface.populateApprove(
+        ownerAddress,
+        tokenAddress,
+        spenderAddress,
+        amount
+      );
 
-      expect(approveResult).toEqual(true);
+      expect(populateApproveResult).toEqual(true);
```

Comment:
https://github.com/yearn/yearn-sdk/pull/283/files#r852590678 so that here we can check the value is more realistic, the tests can be used as documentation and someone seeing a `true` here might expect that that's what that `populateApprove` should return... which is not the case

Reply:
added here https://github.com/yearn/yearn-sdk/pull/283/commits/d6b5daa48ab825a9c68f896cc3b23eb6eb214d24

Reply:
Looks much better, thanks! 🙇